packages/webpack/README.md

```diff
@@ -59,6 +59,20 @@ module.exports = createConfig([
 ])
 ```
 
+#### inspect(object: any, address: ?string):
+
+Prints an object with color highlighting for easier debugging.
+
+Use like this:
+
+```js
+module.exports = createConfig([
+  // ...
+])
+inspect(module.exports) // Print the whole config
```

Comment:
There should be a `const inspect = require('...')` in the snippet 😉

Reply:
Then there should be `require` for `createConfig` in all examples.

Reply:
Ahh, forget it. The snippet is in the list of exports, so it should be clear.